### PR TITLE
Allow binding of ClassLoaderSafeEventListener

### PR DESCRIPTION
**Pull request by Praveen2112**

### CODE REVIEW

**Review by skrzypo987 — APPROVED** [review]
What is the motivation behind it? I believe Inject is implicit if only one constructor is present

**Comment by Praveen2112** [comment]
I guess Inject is implicit only for default constructor

File: lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/classloader/ClassLoaderSafeEventListener.java [hunk 1]
```diff
@@ -27,6 +29,7 @@
     private final EventListener delegate;
     private final ClassLoader classLoader;
 
+    @Inject
     public ClassLoaderSafeEventListener(@ForClassLoaderSafe EventListener delegate, ClassLoader classLoader)
```
Comment (findepi) [inline lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/classloader/ClassLoaderSafeEventListener.java:33]: So `@ForClassLoaderSafe` was not working yet. Do we have other such places?

**Review by findepi — APPROVED** [review]

File: lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/classloader/ClassLoaderSafeEventListener.java (hunk 1 above)
Comment (Praveen2112) [reply lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/classloader/ClassLoaderSafeEventListener.java:33]: Yes..Need to check for others

**Review by lukasz-walkiewicz — APPROVED** [review]

**Review by kokosing — APPROVED** [review]

**Pull request merged**